thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js

```diff
@@ -1,14 +1,17 @@
 import Ember from 'ember';
-import { checkStatus, makeIterable, filterObject } from 'thirdeye-frontend/helpers/utils';
+import { makeIterable, filterObject, toBaselineUrn, filterPrefix } from 'thirdeye-frontend/helpers/utils';
 import EVENT_TABLE_COLUMNS from 'thirdeye-frontend/mocks/eventTableColumns';
-import fetch from 'fetch';
 import config from 'thirdeye-frontend/mocks/filterBarConfig';
 
-//
-// Controller
-//
-
 export default Ember.Controller.extend({
+  entitiesService: Ember.inject.service("rootcause-entities-cache"), // service
```

Comment:
let's use single quotes `'` in javascript please

Reply:
fixed. also, thank you for looking this over. it's a super long PR.